lib/gui/lvgl/Kconfig.memory

```diff
@@ -14,36 +14,24 @@ config LVGL_BITS_PER_PIXEL
 
 choice
 	prompt "Memory pool"
-	default LVGL_MEM_POOL_HEAP_KERNEL
+	default LVGL_MEM_POOL_SYS_HEAP
```

Comment:
how come some Kconfigs are "LV_*" and others are "LVGL_*"?

Reply:
The LV_ come from LVGL itself. LVGL_ are options defined in Zephyr because we still need some.